webpack.config.js

```diff
@@ -30,7 +30,7 @@ const coreConfig = {
             },
             {
                 test: /\.js$/,
-                include: /(node_modules\/engine.io-client|node_modules\/socket.io-client|node_modules\/smartsupp-websocket|node_modules\/device-detector-js)/,
+                include: /(node_modules\/engine.io-client|node_modules\/socket.io-client|node_modules\/smartsupp-websocket|device-detector-js)/,
```

Comment:
Don't even ask me, nemám nejmenší tušení. Jsem vděčnej, že mě taková věc vůbec napadla zkusit, a ještě vděčnější, že to funguje :D